GVRf/Framework/framework/src/main/jni/objects/textures/render_texture_jni.cpp

```diff
@@ -72,6 +76,22 @@ Java_org_gearvrf_NativeRenderTexture_ctorMSAA(JNIEnv* env, jobject obj,
     return reinterpret_cast<jlong>(tex);
 }
 
+JNIEXPORT jlong JNICALL
+Java_org_gearvrf_NativeRenderTexture_ctorWithMonoscopic(JNIEnv* env, jobject obj,
+                                              jint width, jint height, jint sample_count, jint number_views, jboolean monoscopic)
+{
+    int depth_format = DepthFormat::DEPTH_24_STENCIL_8;
+
+    if(number_views > 1) // multiview doesn't work with stencil attachment
+        depth_format = DepthFormat::DEPTH_24;
+    RenderTexture* tex;
+    if(Renderer::getInstance()->isVulkanInstance())
+        tex = static_cast<VulkanRenderer*>(Renderer::getInstance())->createRenderTexture(width, height, sample_count, ColorFormat::COLOR_8888, depth_format , 0, NULL, number_views, monoscopic);
```

Comment:
How is the monoscopic flag being used? Quick search showed no use. If it is not necessary let's remove the flag and this new method.